addons/compat_cup_weapons/compat_cup_weapons_csw/CfgWeapons.hpp

```diff
@@ -104,8 +104,8 @@ class CfgWeapons {
             deployTime = 4;
             pickupTime = 4;
             class assembleTo {
-                ace_csw_kordTripod = "CUP_O_KORD_high_RU";
-                ace_csw_kordTripodLow = "CUP_O_KORD_RU";
+                ace_csw_kordTripod = "CUP_I_KORD_AAF";
+                ace_csw_kordTripodLow = "CUP_I_KORD_high_AAF";
```

Comment:
swap these

Reply:
low tripod doesnt make high weapon, for example

Reply:
Done.